packages/angular-cli/package.json

```diff
@@ -42,26 +42,17 @@
     "compression-webpack-plugin": "^0.3.2",
     "configstore": "^2.0.0",
     "core-js": "^2.4.0",
-    "core-object": "0.0.2",
     "css-loader": "^0.23.1",
     "debug": "^2.1.3",
     "denodeify": "^1.2.1",
     "diff": "^2.2.2",
     "ember-cli-normalize-entity-name": "^1.0.0",
     "ember-cli-preprocess-registry": "^2.0.0",
     "ember-cli-string-utils": "^1.0.0",
-    "enhanced-resolve": "^2.2.2",
     "exists-sync": "0.0.3",
-    "exit": "^0.1.2",
-    "exports-loader": "^0.6.3",
-    "expose-loader": "^0.7.1",
     "extract-text-webpack-plugin": "^2.0.0-beta.4",
-    "file-loader": "^0.8.5",
```

Comment:
Okay I changed the loader in the config to use the full name to avoid this happening again.